app/scripts/modules/cloudfoundry/src/serverGroup/details/sections/BuildSection.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+
+import { ICloudFoundryServerGroupDetailsSectionProps } from './ICloudFoundryServerGroupDetailsSectionProps';
+import { CollapsibleSection } from '../../../../../core/src/presentation';
```

Comment:
```suggestion
import { CollapsibleSection } from '@spinnaker/core';
```

Reply:
This causes errors in IJ for some reason. What is the fix for that?